src/ibex_bluesky_core/callbacks/_fitting.py

```diff
@@ -1,17 +1,19 @@
-"""Creates a readable text file, with comma separated values of Bluesky fitting metrics."""
+"""For IBEX Bluesky scan fitting."""
 
 import csv
 import logging
 import os
+import warnings
 from datetime import datetime
 from pathlib import Path
 from zoneinfo import ZoneInfo
 
 import numpy as np
 from bluesky.callbacks import CallbackBase
+from bluesky.callbacks import LiveFit as _DefaultLiveFit
+from bluesky.callbacks.core import make_class_safe
+from event_model import RunStart, RunStop
 from event_model.documents.event import Event
```

Comment:
You imported `RunStart` and `RunStop` from top level, any reason to not do the same for `Event`?